src/implementations/twig/components/site-header-standardised/site-header-standardised.story.js

```diff
@@ -143,18 +143,27 @@ export const Default = (args) =>
 
 Default.storyName = 'default';
 Default.argTypes = getArgTypes(englishData);
-Default.parameters = { notes: { markdown: notes, json: englishData } };
```

Comment:
but if all the stories in a component have to be displayed "fullscreen" while not adding the parameter in the `export default` instead of repeating it for each story..?

Reply:
done